### PR TITLE
Links

### PR DESCRIPTION
**Pull request by elimelt**
Author: Elijah

## What changes were made?

Pulled all links out into constants.ts and updated footer with real links

### CODE REVIEW

File: src/components/Layout.tsx [hunk 1]
```diff
@@ -41,15 +57,23 @@ interface NavBarProps {
...
 }
 
+const NO_REDIRECT_PATHS = ['/auth', '/join'];
+
 const Layout: React.FC<LayoutProps> = ({ children }) => {
   const { isAuthenticated, isAdmin, loading, member, isVerified } = useAuth();
 
   const navigate = useNavigate();
+  const { pathname } = useLocation();
+  console.log('pathname', pathname);
```
Comment (ShawnCollinge) [inline src/components/Layout.tsx:67]: ?

File: src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+export const LOCAL_API_ENDPOINT = 'http://localhost:8000/'
```
Comment (ShawnCollinge) [inline src/constants.ts:1]: Should this also have no trailing slash?

File: src/components/Layout.tsx (hunk 1 above)
Comment (elimelt) [reply src/components/Layout.tsx:67]: caught lacking

File: src/constants.ts (hunk 2 above)
Comment (elimelt) [reply src/constants.ts:1]: it had a trailing slash earlier so I kept it, but should be fine either way probably. will test rq locally

**Comment by elimelt** [comment]
added no-console rule to linter lol

**Review by ShawnCollinge — APPROVED** [review]

**Pull request merged**